### ISSUE TITLE
ios builds with 2.0.0: Processing node_modules failed. TypeError: Cannot read property 'split' of null

### ISSUE THREAD

**Issue by georgeedwards** [NativeScript/nativescript-cli#1734]
I have just restarted my mac for the first time since I updated to the CLI 2.0.0. My project which was working before now won't build, citing error:

>Processing node_modules failed. TypeError: Cannot read property 'split' of null

I am running OSX El Capitan with XCode 7.3

Even if I run `npm install nativescript -g` I get the same error:

```
Unknown-88-63-df-a3-2e-e9:demo georgeedwards$ npm install nativescript -g
npm WARN excluding symbolic link docs/stylesheets/hightlight.css -> ../../node_modules/highlight.js/src/styles/solarized_light.css
npm WARN deprecated lodash-node@2.4.1: This package has been discontinued in favor of lodash@^4.0.0.
npm WARN deprecated lodash@1.0.2: lodash@<3.0.0 is no longer maintained. Upgrade to lodash@^4.0.0.
npm WARN deprecated graceful-fs@1.2.3: graceful-fs v3.0.0 and before will fail on node releases >= v6.0. Please update to graceful-fs@^4.0.0 as soon as possible. Use 'npm ls graceful-fs' to find it in the tree.
npm WARN excluding symbolic link docs/assets/ir_black.css -> ../../node_modules/highlight.js/src/styles/ir_black.css
npm WARN excluding symbolic link examples/TestFramework/Test Framework.framework/Resources -> Versions/Current/Resources
npm WARN excluding symbolic link examples/TestFramework/Test Framework.framework/Test Framework -> Versions/Current/Test Framework
npm WARN excluding symbolic link examples/TestFramework/Test Framework.framework/Versions/Current -> A
/Users/georgeedwards/.npm-packages/bin/tns -> /Users/georgeedwards/.npm-packages/lib/node_modules/nativescript/bin/nativescript.js
/Users/georgeedwards/.npm-packages/bin/nativescript -> /Users/georgeedwards/.npm-packages/lib/node_modules/nativescript/bin/nativescript.js

> nativescript@2.0.0 postinstall /Users/georgeedwards/.npm-packages/lib/node_modules/nativescript
> node postinstall.js

WARNING: CocoaPods is not installed or is not configured properly.
You will not be able to build your projects for iOS if they contain plugin with CocoaPod file.
To be able to build such projects, verify that you have installed CocoaPods.
Cannot read property 'split' of null
Error: Unknown command 'dev-post-install'. Try '$ tns help' for a full list of supported commands.
    at Object.<anonymous> (/Users/georgeedwards/.npm-packages/lib/node_modules/nativescript/lib/common/errors.js:7:23)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/georgeedwards/.npm-packages/lib/node_modules/nativescript/lib/nativescript-cli.js:8:16)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
/Users/georgeedwards/.npm-packages/lib
└── nativescript@2.0.0 
```

**Comment by georgeedwards** [comment]
[Here](https://gist.github.com/georgeedwards/b3388a408b493cc19eb6a8c5c4ab94c6) is a link to the full failing build stack trace

**Comment by TsvetanMilanov** [comment]
@georgeedwards check if you have CocoaPods installed. If you have it, tell me which version it is.

**Comment by mgol** [comment]
Installing CocoaPods 0.39.0 resolved it for me but before that I've seen it even in the `tns doctor` output which shouldn't happen.

**Comment by georgeedwards** [comment]
Yes, I had to reinstall Cocoapods (with all the El Capitan and Xcode 7.3 quirks) and it started working again. Not sure why it affected `tns doctor`

**Comment by dtopuzov** [comment]
Still valid in master

**Comment by dfg-nota** [comment]
I believe this issue is due to the Cocoapods master repo not having been setup.

It happened to me when I changed to a new user account on my Mac, which of course had an empty ~/.cocoapods directory.
At first tns-doctor reported that master repo was not setup, but then it stopped reporting it without having actually solved the issue.

I fixed it by following the suggestion here: https://github.com/CocoaPods/CocoaPods/issues/4293#issuecomment-188730296
